### ISSUE TITLE
Multi-region deployment: Attempt to join [IP] failed: Error during join version check: dial tcp i/o timeout

### ISSUE THREAD

**Issue by mickhansen** [etcd-io/etcd#765]
**TL;DR** `-peer-election-timeout=500` fixes this issue.

Issue is "fixed" so this is mostly for other users facing the same issues.

I was testing a 3 node multi-region deployment. 2 on US based DigitalOcean, one on Amsterdam based DigitalOcean.

The two first nodes would peer fine but the third node would never be able to peer with the first two (US based nodes).

The EU based node would connect to discovery and get the IP of the US based peers and then start throwing 'Attempt to join [IP] failed: Error during join version check: dial tcp i/o timeout' errors.

A curl call for [IP]:[PORT]/version would complete fine both sides of the atlantic. With a curl call generally taking 180~ ms.

After a few hours of head scratching and help from @philips it finally resulted in trying out the tuning settings - And that apparently worked.

Specifically `-peer-election-timeout=500` worked for me.
